sds_data_manager/constructs/batch_compute_resources.py

```diff
@@ -148,14 +154,50 @@ def __init__(
         # processing_step_name tag. I think this will require
         # setting up a lambda. Maybe there's another way?
         self.job_definition_name = f"fargate-batch-job-definition{processing_step_name}"
+
+        # Get EFS mount information
+        efs_mount_config = scope.node.get_context("efs_mount_config")
```

Comment:
I think you can pass in the efs stack and reference the filesystem information from there. I'm not sure if Batch requires the workaround with exporting/importing the function value but you could get the information that way too rather than needing to put it in the configuration. Something like this:

```
        # NOTE: Workaround to overcome EFS circular dependency when mounting a filesystem
        # https://github.com/aws/aws-cdk/issues/18759
        efs_access_point_id = Fn.import_value("efs-access-pt-id")
        lambda_efs_access = aws_lambda.FileSystem(
            arn=f"arn:aws:elasticfilesystem:{self.region}:{self.account}:access-point/{efs_access_point_id}",
            local_mount_path="/mnt/data",
        )
```

Reply:
Yes. That gives us the access point id. I do that in line 184 and 192. Mine was just reading the config that contains those `import_value`'s name like 'efs-access-pt-id' and other informations as well.